src/components/product-manage/product-inventory.ts

```diff
@@ -88,13 +97,17 @@ class ProductInventory extends Component {
     const quantity = ($quantity.childNodes[0] as HTMLInputElement).value;
 
     const productList = Store.instance.getState().productList;
+    const isEditingSameProduct = (item: ProductItem) => {
+      if (originalName === name && item.name === name) return false;
+      return true;
```

Comment:
아래와 같이 작성하게되면 조금 더 직관적으로 알 수 있을 것 같은데 어떠세요?

```suggestion
      return originalName === name && item.name === name;
```

Reply:
더 깔끔합니다. 수정하겠습니다 :D

Reply:
[f2436bb](https://github.com/woowacourse/javascript-vendingmachine/pull/66/commits/f2436bbd688e87a6537636b0c68b4b5667a14447)
수정했습니다 :D